native/package.json

```
@@ -7,26 +7,38 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo start --web",
-    "test": "jest"
-
+    "test": "jest",
+    "tsc": "tsc",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix"
```

And here if we decide to add it above
```suggestion
    "lint:fix": "eslint . --fix && prettier . --write"
```